plugins/Kaleidoscope-LED-Stalker/src/kaleidoscope/plugin/LED-Stalker.cpp

```diff
@@ -51,9 +52,6 @@ EventHandlerResult StalkerEffect::onKeyEvent(KeyEvent &event) {
   if (::LEDControl.get_mode_index() != led_mode_id_)
     return EventHandlerResult::OK;
 
-  // The simplest thing to do is trigger on both press and release. The color
-  // will fade while the key is held, and get restored to full brightness when
-  // it's released.
   ::LEDControl.get_mode<TransientLEDMode>()->map_[event.addr.toInt()] = 0xff;
```

Comment:
I do feel like, for clarity, if this is setting the key to full brightness on release, even when we don't need it to, it's worth a comment explaining what's going on, so that future hackers don't need to go spelunking too deeply for intent.

Reply:
This comment was added when the API changed such that `onKeyEvent` only fires on press and release, whereas before that change it used to fire continuously while held too. So that comment was added to describe the degraded behaviour it had since that change.

I could certainly add a comment here saying this lights it up on press and release, but isn't that just what `onKeyEvent` means anyway, so wouldn't that be obvious to someone who knows the API?

But what if instead we move the entire `onKeyEvent` handler? I don't think it adds anything. I just tried with it entirely commented out in the cpp and h file and it's working just fine.

Reply:
I added a commit removing the handler. Let me know what you think.

Reply:
If it still works right without the handler, then I approve :)